2200098/exam5.py

```diff
@@ -0,0 +1,13 @@
+# 소득 불균형
+
+T = int(input())
+for t in range(1, T+1) :
+    N = int(input())
+    earn_list = list(map(int, input().split()))
+    aver = sum(earn_list)//N
```

Comment:
```suggestion
    aver = sum(earn_list)/N
```
정확한 평균 값을 구하려면 `//`가 아닌 `/` 연산을 사용하시는 게 맞습니다! 😄